<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScannerClass</class>
 <widget class="QMainWindow" name="ScannerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scanner</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>扫描参数</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>IP范围：</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLineEdit" name="startIP_1"/>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="startIP_2"/>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="startIP_3"/>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="startIP_4"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLineEdit" name="endIP_1"/>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="endIP_2"/>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="endIP_3"/>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>.</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="endIP_4"/>
             </item>
             <item>
              <spacer name="horizontalSpacer_10">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>端口范围：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="startPort">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="endPort">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>线程数：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="threadNum">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="startScanningButton">
           <property name="text">
            <string>开始扫描</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="finishScanningButton">
           <property name="text">
            <string>结束扫描</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QTextBrowser" name="textBrowser"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>startIP_1</tabstop>
  <tabstop>startIP_2</tabstop>
  <tabstop>startIP_3</tabstop>
  <tabstop>startIP_4</tabstop>
  <tabstop>endIP_1</tabstop>
  <tabstop>endIP_2</tabstop>
  <tabstop>endIP_3</tabstop>
  <tabstop>endIP_4</tabstop>
  <tabstop>startPort</tabstop>
  <tabstop>endPort</tabstop>
  <tabstop>threadNum</tabstop>
  <tabstop>startScanningButton</tabstop>
  <tabstop>finishScanningButton</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources>
  <include location="Scanner.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startScanningButton</sender>
   <signal>clicked()</signal>
   <receiver>ScannerClass</receiver>
   <slot>startScanning()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>541</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>123</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>finishScanningButton</sender>
   <signal>clicked()</signal>
   <receiver>ScannerClass</receiver>
   <slot>abort()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>-13</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startScanning()</slot>
  <slot>finishScanning()</slot>
  <slot>abort()</slot>
 </slots>
</ui>
